### DISCUSSION TITLE
ENABLE_SELFHOSTED_MODE - Verify if camera is not reachable by Wyze

### DISCUSSION THREAD

**Discussion by cirdanlunae** [gtxaspec/wz_mini_hacks#396] [General]
I likely am misunderstanding how ENABLE_SELFHOSTED_MODE works. But, from what I understanding, changing this mode to TRUE gives a workaround to the Camera not being able to connect to Wyze's cloud servers, yes? Does it disable Wyze connectivity by default? If it does, how am I 100% sure it can't connect. My goal is to make my Wyze cameras RTSP only with no Wyze telemetry. I've tried SSHing into my Camera and pinging www.wyze.com, and I'm getting a response, making me think that I've not done something right, even with ENABLE_SELFHOSTED_MODE=TRUE

**Comment by virmaior** [comment]
I don't know that much about the topic but wyze doesn't use wyze.com for this. wyze uses amazon web services, so it's irrelevant to block wyze.com

**Comment by Rustymage** [comment]
Block them from the internet and they you're 100% sure they aren't talking to Wyze.
